final/client/src/pages/launches.tsx

```diff
@@ -59,33 +59,23 @@ const Launches: React.FC<LaunchesProps> = () => {
         data.launches.launches.map((launch: any) => (
           <LaunchTile key={launch.id} launch={launch} />
         ))}
-      {data.launches &&
-        data.launches.hasMore && (
-          <Button
-            onClick={() =>
-              fetchMore({
-                variables: {
-                  after: data.launches.cursor,
-                },
-                updateQuery: (prev, { fetchMoreResult, ...rest }) => {
-                  if (!fetchMoreResult) return prev;
-                  return {
-                    ...fetchMoreResult,
-                    launches: {
-                      ...fetchMoreResult.launches,
-                      launches: [
-                        ...prev.launches.launches,
-                        ...fetchMoreResult.launches.launches,
-                      ],
-                    },
-                  };
-                },
-              })
-            }
-          >
-            Load More
-          </Button>
-        )}
+      {data.launches && data.launches.hasMore && (
+        isLoadingMore
+          ? <Loading />
+          : <Button
+              onClick={async () => {
+                setIsLoadingMore(true);
```

Comment:
@hwillson why are you choosing to use `useState` vs fetchMore here?

Reply:
@lifeiscontent can you explain your question a bit more? I'm using state to control a more fine-grained loading indicator, when people click to load more entries using `fetchMore`.